COOKBOOK.md

```diff
@@ -0,0 +1,81 @@
+# Guideline for writing Dojo 2 cookbook recipes
+
+## Purpose
+
+This guideline describes how to create a Dojo 2 cookbook recipe.
+
+Recipes are intended to help consumers find easily consumable solutions to challenges that they are addressing in their projects. Recipes are focused on providing clear, concise steps showing how to accomplish a specific, narrowly defined task. They are almost completely focused on _what_ to do instead of _why_ to do something. If additional explanation is required, links should be provided to relevant documentation for consumers that wish to learn more about a specific topic.
+
+## Summary
+
+The following rules should be followed when creating a cookbook recipe. See the [discussion](#discussion) section for more information about each point.
+
+- [ ] [Contains proper front matter](#contains-proper-front-matter)
+- [ ] [Contains a title heading](#contains-a-title-heading)
+- [ ] [Contains brief Objective section](#contains-brief-objective-section)
+- [ ] [Properly formated Procedure section](#properly-formated-procedure-section)
+- [ ] [Contains list of additional resources](#contains-list-of-additional-resources)
+- [ ] [No boiler plate steps](#no-boiler-plate-steps)
+- [ ] [Proper casing for titles](#proper-casing-for-titles)
+- [ ] [Proper voice and tone use (brief, concise and to the point)](#proper-voice-and-tone-use)
+- [ ] [No unnecessary contractions or slang](#no-unnecessary-contractions-or-slang)
+- [ ] [No pronouns or, if used, have an obvious meaning to non-native English speakers](#no-pronouns-or-if-used-have-an-obvious-meaning-to-non-native-english-speakers)
+- [ ] [No abbreviations](#no-abbreviations)
+- [ ] [Acronyms are expanded on first use](#acronyms-are-expanded-on-first-use)
+- [ ] [Spell-checked against US-English dictionary](#spell-checked-against-us-english-dictionary)
```

Comment:
Might want to add one here and elsewhere that I've seen lately, which is to not use the word file to describe a resource, since the reader may be using the web editor. Also, don't use the word folder, call it a directory which to me feels like the more cross-platform agnostic term.